.travis-install.sh

```diff
@@ -32,9 +32,9 @@ if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
   PROTOC_URL=https://github.com/google/protobuf/releases/download/v3.5.1/protoc-3.5.1-osx-x86_64.zip
 else
   # Install swift
-  SWIFT_URL=https://swift.org/builds/swift-4.1.1-release/ubuntu1404/swift-4.1.1-RELEASE/swift-4.1.1-RELEASE-ubuntu14.04.tar.gz
```

Comment:
I would prefer if we could have CI run with both Swift 4.2 and Swift 5, but I don't see a good way of achieving that besides a lot of duplication. What do you think?

Reply:
It's possible to setup a matrix of travis builds that have separate install commands, https://github.com/kylef/PathKit/blob/73f8e9dca9b7a3078cb79128217dc8f2e585a511/.travis.yml then you could pass the version to this script

Reply:
Nice, looks like your suggestion is working @keith 👍